compiler/rustc_trait_selection/src/traits/error_reporting/mod.rs

```diff
@@ -2963,19 +2965,26 @@ impl<'tcx> InferCtxtPrivExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
                     rustc_transmute::Reason::DstIsPrivate => format!(
                         "`{dst}` is or contains a type or field that is not visible in that scope"
                     ),
-                    // FIXME(bryangarza): Include the number of bytes of src and dst
                     rustc_transmute::Reason::DstIsTooBig => {
                         format!("The size of `{src}` is smaller than the size of `{dst}`")
                     }
+                    rustc_transmute::Reason::DstHasStricterAlignment => {
+                        format!(
+                            "The alignment of `{src}` should be stricter than that of `{dst}`, but it is not"
```

Comment:
```suggestion
                            "The minimum alignment of `{src}` should be greater than that of `{dst}`, but it is not"
```

It'd be cool to have `note:`s that said exactly what the minimum alignments of src and dst are, but that can wait for a future PR.